gem/management/commands/remove_empty_nav_tags.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import BaseCommand
+from molo.core.models import ArticlePage
+
+
+class Command(BaseCommand):
+    help = 'Remove empty nav_tags from all articles'
+
+    def handle(self, **options):
+        for article in ArticlePage.objects.all():
+            for nav_tag in article.nav_tags.all():
+                if nav_tag.tag is None:
+                    nav_tag.delete()
+
+            if article.live:
+                article.save_revision().publish()
+            else:
+                article.save_revision()
```

Comment:
I'm not suuuper happy that this is saving every article, regardless of if any tags were actually changed or not.